pkg/xen-tools/initrd/base.files

```diff
@@ -1,4 +1,8 @@
 /bin/busybox
+/sbin/mke2fs
+/lib/libext2fs.so.2*
+/lib/libcom_err.so.2*
+/lib/libe2p.so.2*
```

Comment:
You were right, the musl library was already present in the initrd. The missing libraries had to be specified explicitly. But, one way or another, this allowed me to reduce the size of initrd from 1.7 MB to 1.2 MB.